libs/dal/src/types/mongo.ts

```diff
@@ -0,0 +1,69 @@
+import { Condition } from 'mongoose';
```

Comment:
Stricter overrides for all the `mongoose` generics, linked in the `@see` directive.